run/test/specs/group_tests_leave_group.spec.ts

```diff
@@ -21,36 +24,24 @@ async function leaveGroupIos(platform: SupportedPlatformsType) {
   await createGroup(platform, device1, userA, device2, userB, device3, userC, testGroupName);
   await device3.clickOnByAccessibilityID('More options');
   await sleepFor(1000);
-  await device3.clickOnByAccessibilityID('Leave group');
+  await device3.clickOnElementAll(new LeaveGroupButton(device3));
+  // Modal with Leave/Cancel
   await device3.clickOnByAccessibilityID('Leave');
   await device3.navigateBack(platform);
   // Check for control message
-  await device2.waitForControlMessageToBePresent(`${userC.userName} left the group.`);
-  await device1.waitForControlMessageToBePresent(`${userC.userName} left the group.`);
-  await closeApp(device1, device2, device3);
-}
-// TO FIX (LEAVE GROUP CONFIRMATION ON DIALOG NOT WORKING)
-async function leaveGroupAndroid(platform: SupportedPlatformsType) {
-  const testGroupName = 'Leave group';
-  const { device1, device2, device3 } = await openAppThreeDevices(platform);
-  // Create users A, B and C
-  const [userA, userB, userC] = await Promise.all([
-    newUser(device1, 'Alice', platform),
-    newUser(device2, 'Bob', platform),
-    newUser(device3, 'Charlie', platform),
-  ]);
+  const groupMemberLeft = localize('groupMemberLeft')
+    .withArgs({ name: userC.userName })
+    .strip()
+    .toString();
 
-  // Create group with user A, user B and User C
-  await createGroup(platform, device1, userA, device2, userB, device3, userC, testGroupName);
-  await device3.clickOnByAccessibilityID('More options');
-  await sleepFor(1000);
-  await device3.clickOnTextElementById(`network.loki.messenger:id/title`, 'Leave group');
-  await device3.clickOnElementAll({
+  await device1.waitForControlMessageToBePresent(groupMemberLeft as ControlMessage);
+  await device2.waitForControlMessageToBePresent(groupMemberLeft as ControlMessage);
```

Comment:
same comment with `as`

Reply:
@burtonemily Not sure what Audric's comment was but if it's the same as mine then it's just moving the as cast to the variable declartion and not do it when calling the function.